CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # [Unreleased](https://github.com/pybamm-team/PyBaMM/)
 
+## Bug Fixes
+
+- Fixed a bug where if the first step(s) in a cycle are skipped then the cycle solution started from the model's initial conditions instead of from the last state of the previous cycle ([#3708](https://github.com/pybamm-team/PyBaMM/pull/3708))
```

Comment:
if this is going into 24.1 this needs to move

Reply:
what’s the process? should i just manually make a new section for the next release candidate?

Reply:
You can leave it as it is. The workflow will automatically create a v24.1rc1 section with the new changes.

Reply:
cool. where/when should this be merged?